routes.js

```diff
@@ -39,7 +40,6 @@ router.get("/getWorkerDetail", async (req, res) => {
   }
 });
 
-
 router.get("/getWorkerDetail/:id", async (req, res) => {
   try {
     const workers = await Worker.findById(req.params.id);
```

Comment:
@onetario , What suppose if there is no id in the query means?, why error is not handled